src/app/App.tsx

```diff
@@ -3,6 +3,9 @@ import { Provider } from "react-redux";
 import { store } from "app/store";
 import { Home, LoginScreen, ProfilePage } from "screens";
 import { ProtectedRoute, OnLoad } from "features/auth/components";
+import axios from "axios";
+
+axios.defaults.withCredentials = true;
```

Comment:
This feels a little wrong to me just because you're doing this in app/App.tsx.
What do you think about creating a file in shared/utils/axios/index.ts that
 imports axios 
 axios.defaults.withCredentials = true;
export default axios;

And then import axios from this file in the other files

Reply:
We can also scope the potential future configurations to this file

Reply:
This also reduces potential bugs in the app. We may have a bunch of default configurations that we may not know about.